src/click.c

```diff
@@ -311,7 +292,8 @@ static int route_click(Con *con, xcb_button_press_event_t *event, const bool mod
     else if ((dest == CLICK_BORDER || dest == CLICK_DECORATION) &&
              is_left_or_right_click) {
         DLOG("Trying to resize (tiling)\n");
-        tiling_resize(con, event, dest);
+        tree_render();
```

Comment:
Ok, lets add an explanation:

```suggestion
        /* Consider this layout: V[w1 T[w2 w3]], w2 is active.
         * User clicks and drags w3. Since we updated the tree (con_activate()
         * above) we need to re-render the tree before returning to the event
         * loop (drag_pointer inside tiling_resize runs its own event-loop). */
        tree_render();
```